modules/search/src/main/java/com/sygic/maps/module/search/SearchFragment.kt

```diff
@@ -114,8 +148,12 @@ class SearchFragment : Fragment(), SdkInitializationManager.Callback {
         super.onCreate(savedInstanceState)
 
         fragmentViewModel = ViewModelProviders.of(this, viewModelFactory)[SearchFragmentViewModel::class.java]
+        searchToolbarViewModel = ViewModelProviders.of(this, viewModelFactory.with(searchToolbarInitComponent))[SearchToolbarViewModel::class.java].apply {
+            this.keyboardVisibilityObservable.observe(this@SearchFragment, Observer<Boolean> { if (it) showKeyboard() else hideKeyboard() })
```

Comment:
zobrazuje a skryva to sice klavesnicu, ale to je asi skor iba dosledok. Nesedlo by viac nieco ako `focusChanged.observe` ?

Reply:
může být, jen vysunutí klávesnice reálně focus nezajistí.